influunt-app/test/spec/controllers/login.js

```diff
@@ -45,22 +47,29 @@ describe('Controller: LoginCtrl', function () {
     expect(form.$valid).toBe(false);
   });
 
-  it('caso o login seja executado com sucesso, deve ser redirecionado para tela inicial do sistema',
-    inject(function($state) {
-      var usuario = {
-        id: 'teste',
-        login: 'teste',
-        email: 'teste@example.com',
-        root: true,
-        permissoes: []
-      }
-      $httpBackend.expectPOST('/login').respond(usuario);
-
-      scope.submitLogin(true);
-      $httpBackend.flush();
-
-      expect($state.current.name).toBe('app.main');
-    }));
+  describe('caso o login seja executado com sucesso', function() {
+    beforeEach(function(done) { done(); });
+    it('caso o login seja executado com sucesso, deve ser redirecionado para tela inicial do sistema',
+      function(done) {
```

Comment:
describe mais it com repetindo os texto. 'caso o login seja executado com sucesso' no describe e ele repete do it.